lib/touchbar.es

```diff
@@ -92,13 +92,8 @@ export const touchBar = new TouchBar({
   escapeItem: poibutton,
 })
 //Change Volume btn
-export const touchBarreinit = (muted) => {
-  if (muted) {
-    volume.icon = path.join(ROOT, 'assets', 'img', 'touchbar', 'volume-off.png')
-  }
-  else {
-    volume.icon = path.join(ROOT, 'assets', 'img', 'touchbar', 'volume-up.png')
-  }
+export const touchBarreinit = () => {
```

Comment:
Well, I don't think that is good enough, if I wrote that, I would have named it refreshGamePage.